frontend/src/components/grantaccess/form/RoleSelect.vue

```diff
@@ -0,0 +1,62 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import { ErrorMessage, Field } from 'vee-validate';
+import Dropdown from 'primevue/dropdown';
+import type { FamApplicationRole } from 'fam-app-acsctl-api';
+import type { FamRoleDto } from 'fam-admin-mgmt-api/model';
+
+const props = defineProps({
+    roleId: { type: Number, default: 0 },
+    roleOptions: {
+        type: [Array<FamApplicationRole[]>, Array<FamRoleDto[]>],
```

Comment:
oh sorry also here, it should be` [Array<FamApplicationRole>, Array<FamRoleDto>]`

Reply:
Thanks, I am going to update it.